document/js-api/index.bs

```diff
@@ -1095,28 +1097,23 @@ The algorithm <dfn>ToWebAssemblyValue</dfn>(|v|, |type|) coerces a JavaScript va
 1. If |type| is [=f64=],
     1. Let |f64| be ? [=ToNumber=](|v|).
     1. Return [=f64.const=] |f64|.
-1. Assert: |type| is a reference type.
-1. If |type| is [=externref=],
-    1. Do nothing.
 1. If |type| is [=funcref=],
-    1. If |v| is not an [=Exported function=] or null, throw a {{TypeError}}.
-1. Return the result of [=allocating a host address=] for |v|.
-
-</div>
-
-<div algorithm>
-    For <dfn>allocating a host address</dfn> for a value |v|, perform the following steps:
     1. If |v| is null,
-        1. Return [=ref.null=].
+        1. Return [=ref.null=] [=funcref=].
```

Comment:
Isn't it `ref.null func`, or did that change?

Reply:
In the spec's internal AST it's (currently) funcref, to avoid introducing another syntactic class. However, that will change with the funcref proposal. In the text format, it's already func, for forward compat.